cpu/gd32v/include/periph_cpu.h

```diff
@@ -33,7 +33,26 @@ extern "C" {
  * @name    Power management configuration
  * @{
  */
-#define PROVIDES_PM_SET_LOWEST
+/**
+ * @brief   Number of usable low power modes
+ */
+#define PM_NUM_MODES    (3U)    /**< Number of usable low power modes */
+
+/**
+ * @brief   Power modes
+ */
+enum {
+    GD32V_PM_STANDBY = 0,       /**< STANDBY mode,  */
+    GD32V_PM_DEEPSLEEP = 1,     /**< DEEPSLEEP mode, corresponds to STOP mode of STM32 */
+    GD32V_PM_IDLE = 2           /**< IDLE mode */
+};
+
+/**
+ * @brief   Wake-up pin used
```

Comment:
How does this work?
Can this be any pin? Can there only be one?

Reply:
This is reused by STM32 and refers to the PA0/WKUP pin that all STM32 MCUs and the GD32V MCU have. That means it can be used or not, but it is not configurable. Theoretically, it should also be possible to use `WFE` instead of `WFI` for deep sleep and standby modes to wake up the MCU when toggling pins for which the EXTI line is configured for events. But we don't have a corresponding configuration infrastructure for this yet.

Reply:
Ah so this would be 

> Set this to 1 to enable wake from Deep Sleep via PA0/WKUP

Ideally this would be a proper `CONFIG_` define (also for STM32), but that could be done as a follow up.
A comment will already help a lot.